locations/items.py

```diff
@@ -29,6 +40,49 @@ class GeojsonPointItem(scrapy.Item):
     ref = scrapy.Field()
     brand = scrapy.Field()
     brand_wikidata = scrapy.Field()
-    located_in = scrapy.Field()
```

Comment:
I've got nothing against `image` being added, but I don't think `located_in` should have been removed. Mistake when merging?

Reply:
I thought that would prompt a discussion. I think you will only want to be expending effort on "located in" coding for reasonably sized brands. That kind of implies a wikidata code, or certainly that one would be created if not already in existence. In which case why have the ambiguity of name and code? Having said that I will add back in further commit on branch unless you agree with my reasoning.

Reply:
Urm, well having `image` twice is defiantly wrong.

For `located_in`, I can see your point, but I'm not sure.

Reply:
ouch ... I missed the double 'image' .. to be fixed also

Reply:
Ok so I have pushed another commit: duplicate image member removed, located_in re-instated, has_geo() check in calires spider removed.

On the latter change it is a general thing that some spiders will wish to remove pages without geo. A piece of pipeline code that looks for "remove_missing_geo_items" member in a spider may be of use but I will not overload my current commit!